scala/lambdas/ingest-mapper/src/main/scala/uk/gov/nationalarchives/ingestmapper/MetadataService.scala

```diff
@@ -88,7 +88,26 @@ class MetadataService(s3: DAS3Client[IO], discoveryService: DiscoveryService) {
           Stream.evals {
             topLevelIdsToDepartmentSeries.map { itemToDepartmentSeries =>
               val parentPaths = getParentPaths(json, itemToDepartmentSeries)
```

Comment:
It's `Map[UUID, String]` so `idToParent` makes sense. You know it isn't a full parent entity because it's a String. Again, I'd like to keep the variable names shorter and infer meaning from type signatures and context.